package.json

```diff
@@ -22,8 +23,8 @@
         "@types/react-router-dom": "^5.1.7",
         "@types/reactstrap": "^8.7.1",
         "@types/styled-components": "^5.1.9",
-        "@typescript-eslint/parser": "^4.20.0",
-        "@typescript-eslint/eslint-plugin": "^4.20.0",
+        "@typescript-eslint/eslint-plugin": "^4.19.0",
+        "@typescript-eslint/parser": "^4.19.0",
```

Comment:
Looks like this PR downgrades some NPM packages that has been updated. You can fix these manually. Just take the newest one here and paste them in, and do npm install to make sure it works